src/containers/LoginFormContainer.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from "react";
+import FacebookLogin from "react-facebook-login";
+import GoogleLogin from "react-google-login";
+import { BASE_URL, API_FB_ID, API_GOOGLE_ID } from "../constants";
+import axios from "axios";
+
+export default class LoginFormContainer extends Component {
+  state = {
+    isLoggedIn: false,
+    userID: "",
+    firstName: "",
+    lastName: "",
+    email: "",
+    picture: "",
+    token: ""
+  };
+
+  componentClicked = () => console.log("clicked");
+
+  responseFacebook = response => {
+    console.log(response);
+
+    this.setState({
+      isLoggedIn: true,
+      userID: response.userID,
+      firstName: response.first_name,
+      lastName: response.last_name,
+      email: response.email,
+      picture: response.picture.data.url,
+      token: response.accessToken
+    });
+
+    console.log(response.accessToken);
+  };
+
+  responseGoogle = response => {
+    console.log(response);
+
+    this.setState({
+      isLoggedIn: true,
+      userID: response.profileObj.googleId,
+      firstName: response.profileObj.givenName,
+      lastName: response.profileObj.familyName,
+      email: response.profileObj.email,
+      birthday: response.birthday,
+      picture: response.profileObj.imageUrl
+    });
+  };
+
+  addNewUser = () => {
+    axios
+      .post(`${BASE_URL}/login`, {
+        firstName: this.state.first_name,
+        lastName: this.state.last_name,
+        email: this.state.email,
+        picture: this.state.picture.data.url,
+        token: this.state.accessToken
+      })
+      .then(() => {
+        this.props.history.push("/");
+      })
+      .catch(err => console.log(err));
```

Comment:
@jskurzynska is there any difference between using .catch() and just handling promise reject in .then()? I used the latter usually, but I believe it's up to our agreement which one should be used.